### ISSUE TITLE
Emulate raspvisor with inside QEMU

### ISSUE THREAD

**Issue by deepsthewarrior** [matsud224/raspvisor#1]
Thanks for the Nice work on Raspvisor :)
Have you tried emulating raspvisor inside QEMU with the echo.bin, test.bin, mini-os that you provided in the repo?

**Comment by matsud224** [comment]
No. I only tried running raspvisor on a real Raspberry Pi 3 machine.2023/04/14 22:52、Deepika Rani Kaliappan Mahalingam ***@***.***>のメール:﻿
Have you tried emulating raspvisor inside QEMU with the echo.bin, test.bin, mini-os that you provided in the repo?
